tmpl/wiki/tutorial-lwt.md

```diff
@@ -60,11 +99,9 @@ Add `foo.ml` with the following content and edit it:
   module Main (C : CONSOLE) = struct
     let start c =
       (* the guts go here *)
-      return ()
   end
 ```
 
-
 Assuming you are building for POSIX using native kernel sockets, compile the application by:
```

Comment:
I think "using native kernel sockets" doesn't add anything and potentially confuses the reader.  "building for POSIX" could also probably be reworded to something a bit more accessible, although I'm having trouble thinking of something that's not too wordy.